docs/topics/installing-web-console-on-openshift.adoc

```diff
@@ -41,7 +41,7 @@ endif::[]
 ifdef::ocp-41,ocp-311[]
 .Procedure
 
-. Download the installation archive file from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} download page].
+. Navigate to the  link:{MTADownloadPageURL}[{ProductShortName} Download page] and download the installation archive file.
```

Comment:
Be a bit more specific than "installation archive file". 

"{ProductShortName} installation file"